library/Freckle/App/Kafka/Consumer.hs

```diff
@@ -203,7 +206,9 @@ runConsumer pollTimeout onMessage =
 
   onFinish finishResult = do
     consumer <- view kafkaConsumerL
-    logExMay "Unable to commit offsets" $ commitAllOffsets OffsetCommit consumer
+    logExMay
+      "Unable to commit offsets"
+      (silenceNoOffsetError <$> commitAllOffsets OffsetCommit consumer)
```

Comment:
Told ya?

Reply:
mhmm.... It's pretty sweet that it pointed us to this issue